### DISCUSSION TITLE
Table entry shown as a link

### DISCUSSION THREAD

**Discussion by arkansas-coder** [requarks/wiki#6063] [Help / Questions] [answered]
Using a simple table
```
| header |
| --- |
| example.com |
```
creates `example.com` as a link rather than just text. Why does this happen, and can it be stopped?

**Comment by fnafox** [comment, marked as answer]
Replace the period with HTML code `&#46;` like so:
```
| test |
| --- |
| example&#46;com |
```

**Reply by arkansas-coder** [reply]
Thanks! Problem solved
